src/fmtargs.h

```diff
@@ -1,48 +1,173 @@
 /*
- * Copyright (c) 2023, Viktor Soderqvist <viktor.soderqvist@est.tech>
+ * Copyright Redis Contributors.
  * All rights reserved.
  * SPDX-License-Identifier: BSD 3-Clause
  *
  * To make it easier to map each part of the format string with each argument,
  * this file provides a way to write
  *
- *     printf("xxx %s yyy %s zzz %s end\n", arg1, arg2, arg3);
+ *     printf("a = %s, b = %s, c= %s\n",
```

Comment:
can't help it, my eyes are attracted to the wrong things..
```suggestion
 *     printf("a = %s, b = %s, c = %s\n",
```

Reply:
Yeah I saw it too after pushing. :) I'm very sensitive to these too. It's a needle in the eye.